packages/application-generic/src/services/bull-mq/bull-mq.service.ts

```diff
@@ -18,7 +18,7 @@ import {
   InMemoryProviderEnum,
   InMemoryProviderService,
 } from '../in-memory-provider';
-import { validateMemoryDbClusterProviderConfig } from '../in-memory-provider/providers/memory-db-cluster-provider';
+import { BulkJobOptions } from 'bullmq/dist/esm/interfaces';
```

Comment:
The import should be done from the package. Try to import it in the line 13.

Reply:
Good catch! missed this one!